rotkehlchen/data_import/importers/binance.py

```diff
@@ -294,11 +304,12 @@ def process_trades(
                 to_amount is None or to_amount == ZERO or
                 from_amount is None or from_amount == ZERO
             ):
-                log.warning(
-                    f'Skipped binance rows {data} because '
-                    f"it didn't have enough data",
-                )
-                importer.db.msg_aggregator.add_warning("Skipped some rows because couldn't find amounts or it was zero")  # noqa: E501
+                for row in trade_rows:
+                    importer.log_warning(
+                        row_index=row[INDEX],
+                        csv_row=row,
+                        msg='Trade associated with row is missing required data.',
```

Comment:
previous message told that what is missing, now it only says 'data'. Is this info sent anywhere else?